common/interfaces/user_interface/ui_multi_instruction.h

```diff
@@ -13,11 +13,23 @@
 #include "ui_common.h"
 
 /// Maximum number of instruction screens
-#define MAX_NUM_OF_INSTRUCTIONS 7
+#define MAX_NUM_OF_INSTRUCTIONS 10
 
 /// Maximum number of character in a single instruction screen
 #define MAX_NUM_OF_CHARS_IN_AN_INSTRUCTION 85
 
+/**
+ * @brief 
+ * 
+ */
+typedef struct{
```

Comment:
by looking at this structure, it seems that a single screen can accommodate both text and image.. is it true?

if it can only store one of the information then we should use `union`

Reply:
A single object can contain both text and image, was used earlier but we removed the FCC id text and added it to the image.